src/styles/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $width-calendar;
+  gap: $gap;
+  padding: $padding-calendar;
+
+  $start-days:
+    $mon mon,
+    $tue tue,
+    $wed wed,
+    $thu thu,
+    $fri fri,
+    $sat sat,
+    $sun sun;
+
+  @each $day, $start-day in $start-days {
+    &--start-day--#{$start-day} :first-child {
+      margin-left: $day;
+    }
+  }
+
+  @for $days from 28 through 31 {
+    &--month-length-#{$days} :nth-child(n + #{$days + 1}){
+      display: none;
+    }
+  }
+
+  &__day {
+    border: 1px solid black;
+    background-color: $background-color;
+    height: $size-day-cell;
+    width: $size-day-cell;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @for $num from 1 through 31 {
+      &:nth-child(#{$num})::before {
+      content: "#{$num}";
+      font-family: Arial,sans-serif;
+      font-size: 30px;
```

Comment:
Consider applying the font family and the font size to the before element instead of generating these styles for each child in the cycle